examples/layer-browser/src/data-samples.js

```diff
@@ -135,26 +135,64 @@ export function getPointCloud() {
   return _pointCloud;
 }
 
+const sfBoundingBox = [-122.6, 37.6, -122.2, 37.9];
```

Comment:
Nit: We use all CAPS for constants: `const SF_BOUNDING_BOX = `

Reply:
fixed